stake-pool/js/test/instructions.test.ts

```diff
@@ -156,7 +148,7 @@ describe('StakePoolProgram', () => {
         if (pubKey == stakePoolAddress) {
           return stakePoolAccount;
         }
-        if (pubKey.equals(CONSTANTS.poolTokenAccount)) {
+        if (pubKey.toBase58() == '9q2rZU5RujvyD9dmYKhzJAZfG4aGBbvQ8rWY52jCNBai') {
```

Comment:
nit: to be totally sure, let's `===` for all of these
```suggestion
        if (pubKey.toBase58() === '9q2rZU5RujvyD9dmYKhzJAZfG4aGBbvQ8rWY52jCNBai') {
```